frontend/src/components/VHideButton.vue

```diff
@@ -0,0 +1,11 @@
+<template>
+  <VButton
+    variant="transparent-gray"
+    size="large"
+    class="label-bold inline-flex gap-1 md:right-8"
```

Comment:
Is the gap value here consistent with Figma mockups? 
Normally, if you add `has-icon-end`, `VButton` should handle the gap correctly, setting it to 1 for "small" size buttons, and to 2 for "medium" and "large" ones.

Reply:
I'll switch to that and confirm it matches the mockups; thanks!